.circleci/config.yml

```diff
@@ -77,8 +77,11 @@ jobs:
           command: pip install -e . --install-option="--buildgo"
           name: setup
       - run:
-          command: pytest ./tests
+          command: pytest --cov=hivemind ./tests
           name: tests
+      - run:
+          command: codecov
+          name: codecov
```

Comment:
Better to run this for all versions just in case (codecov should be able to merge the reports)

Reply:
👌